json/schema/full_import.schema.json

```diff
@@ -247,6 +247,12 @@
             "type": "string",
             "maxLength": 255,
             "description": "Indicates whether the user should be accessing by SSO or not"
+          },
+          "roles": {
+            "type": "string",
+            "maxLength": 255,
+            "pattern": "^(admin|super_admin|user|,)*$",
```

Comment:
This would be ideal as an array in the input with enum, but we have to accept a string (since it will come from a CSV primarily) like `"admin, user"` for example, can't use enum for that.